articles/how-to-setup-ci-cd-to-vue-with-firebase-hosting-and-github-actions/index.md

```diff
@@ -0,0 +1,223 @@
+
+![github actions workflow files](10-github-actions-workflow-files.jpg)
+
+The major objective of these two workflows is to build the application and deploy it to Firebase Hosting automatically. 
+
+The first workflow file **firebase-hosting-merge.yml** will execute in every merge to the master branch. It will deploy the merge to our production or live website environment. 
+
+The second workflow file **firebase-hosting-pull-request.yml** will only execute when a pull-request is created from any arbitrary branch to the master branch. It will deploy the push to a separate URL which will enable us to test out the result before executing the merge to the pull request.
+
+Now, with these two workflow scripts running, we won't have to deal with deployment every time we make changes to the code and push to the GitHub repo. We only need to push and merge the changes to the master branch. The rest of the build and deployment process will be automatically taken care of by Firebase and GitHub will automatically execute the deployment. 
+
+### Step 4: Testing to see if GitHub actions works
+
+Now that we have everything set up, let's test it out. Since we have two workflow files already in our Vue project directory, we already have some changes to push. 
+
+Now, we need to create a new **Git branch** in the root project and push the branch to the GitHub repo.
+
+Now from the GitHub console, we can navigate to that branch and create a **pull request** in GitHub from the created branch to the master branch, which will automatically trigger the workflow with Github actions as shown in the screenshots below:
+
+![testing deployment result](11-testing-deployment-result-1.png)
+
+
+This proves that our configuration is working properly. Now after accepting the pull-request and merging the branch to the master branch will automatically trigger deployment as well.
+
+We can check the successful deployment action from the [GitHub repo actions](https://github.com/Worawat101/vue_cicd_example/runs/1760954615?check_suite_focus=true). 
+
+The changes will be deployed live in the hosted website [demo here](https://vue-cicd-b28ff.web.app/).
+
+### Recap
+
+Well, the overall objective of the article was to get ourselves familiar with CI/CD  with Vue.js project, Firebase, and GitHub actions workflows. This tutorial article delivered the detailed stepwise concepts of how to configure the CI/CD process. The major step revolved around the configuration of Firebase Hosting with GitHub workflows. The steps are simple and easy to understand and apply. All the manual work for iteratively building and deploying to the live server is nullified by this process. The only work we have to do is to push our changes and merge the changes to the master branch. The rest of the operation is automatically handled by Firebase and GitHub action scripts. 
```

Comment:
It would sound better if you excluded the well at the beginning so that the block starts with :
_The overall_

Is this statement really necessary?

>  This tutorial article delivered the detailed stepwise concepts of how to configure the CI/CD process. The major step revolved around the configuration of Firebase Hosting with GitHub workflows. The steps are simple and easy to understand and apply.